src/inference/src/dev/core_impl.cpp

```diff
@@ -737,13 +757,16 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::compile_model(const std::shared_ptr<
     // if auto-batching is applicable, the below function will patch the device name and config accordingly:
     auto model = apply_auto_batching(model_, deviceName, config_with_batch);
 
-    auto parsed = parseDeviceNameIntoConfig(deviceName, config_with_batch, is_proxy_device(device_name));
+    auto local_core_config = coreConfig;
+    auto parsed = parseDeviceNameIntoConfig(deviceName, config_with_batch, is_proxy_device(deviceName));
```

Comment:
maybe we need to extend `parsed` structure? so `parseDeviceNameIntoConfig` will return `localCore` as well:
```c++
// usage like
auto cacheManager = parsed.core_config.get_cache_config_for_device(plugin)._cacheManager
```

Reply:
It will requires to extract `CoreConfig` from `CoreImpl` to be possible to use it.

Reply:
The `parseDeviceNameIntoConfig` has been updated to return local core configuration.